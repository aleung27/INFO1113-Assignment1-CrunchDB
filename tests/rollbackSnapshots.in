SET s 9
SNAPSHOT
SET h 7 3 5 1
SNAPSHOT
SORT h
LIST ENTRIES
ROLLBACK 2
REV h
LIST ENTRIES
ROLLBACK 2
LIST ENTRIES
DEL s
LIST ENTRIES
ROLLBACK 1
LIST SNAPSHOTS
LIST ENTRIES
