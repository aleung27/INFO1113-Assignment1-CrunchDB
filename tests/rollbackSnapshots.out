> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> h [1 3 5 7]
s [9]

> ok

> ok

> h [1 5 3 7]
s [9]

> ok

> h [7 3 5 1]
s [9]

> ok

> h [7 3 5 1]

> ok

> 1

> s [9]

